src/app/perspective-status.component.ts

```diff
@@ -389,24 +389,22 @@ export class PerspectiveStatus implements OnChanges, AfterViewInit, AfterViewChe
     }
   }
 
-  /**
-   * Updates the gradient color scale for the shape based on the
-   * scoreThresholds.
-   */
-  updateGradient() {
-    // The number of points to use to calculate the gradient.
-    let gradientPointCount = 100;
-    const sliderGradient = new toxicLibsJS.color.ColorGradient();
+  getFirstGradientRatio(): number {
```

Comment:
Why is this a function instead of just an attribute?

Reply:
I wanted to keep the value as a constant but also provide a way to get it in the test.